extensions/tiled-assets/examples/example-dimension.json

```diff
@@ -0,0 +1,234 @@
+{
+    "stac_version": "0.8.0",
+    "stac_extensions": [
+        "eo",
+        "tiled-assets"
+    ],
+    "id": "sen2l2a-cog-converted",
+    "type": "Feature",
+    "bbox": [
+        14.326171874999998, 48.603515625, 14.501953125000002, 48.33984374999999
+    ],
+    "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+            [
+                [
+                    14.326257705688475,
+                    48.33992958068847
+                ],
+                [
+                    14.501867294311525,
+                    48.33992958068847
+                ],
+                [
+                    14.501867294311525,
+                    48.60342979431152
+                ],
+                [
+                    14.326257705688475,
+                    48.60342979431152
+                ],
+                [
+                    14.326257705688475,
+                    48.33992958068847
+                ]
+            ]
+        ]
+    },
+    "properties": {
+        "datetime": "2018",
```

Comment:
This is not a valid date-time, must conform to RFC3339 date-time.
I guess you should use the datetime-range extension here.

Reply:
Done. I cannot find the datetime range extension anymore, what has happened to it?

Reply:
It is part of the core, see https://github.com/radiantearth/stac-spec/blob/master/item-spec/common-metadata.md#date-and-time

Reply:
Thanks! (It was in the extensions, so I was confused). I updated the examples to also incorporate the datetime range.